oak_runtime/src/node/http/mod.rs

```diff
@@ -480,11 +481,27 @@ impl Pipe {
     }
 }
 
+// HTTP requests can either provide JSON formatted labels or protobuf encoded labels. But exactly
+// one of these should be provided. This method checks that exactly one label is provided in a
+// header in the request and extracts it for use for further handling of the request.
 fn get_oak_label(req: &HttpRequest) -> Result<Label, OakStatus> {
-    match req.headers.get(oak_abi::OAK_LABEL_HTTP_KEY) {
-        Some(label_str) => parse_json_label(label_str.to_vec()),
-        None => {
-            warn!("No HTTP label found.");
+    let headers = (
+        req.headers.get(oak_abi::OAK_LABEL_HTTP_JSON_KEY),
+        req.headers.get(oak_abi::OAK_LABEL_HTTP_PROTOBUF_KEY),
+    );
+
+    match headers {
+        (Some(json_label), None) => parse_json_label(json_label.to_vec()),
+        (None, Some(protobuf_label)) => Label::decode(&protobuf_label[..]).map_err(|err| {
+            warn!("Could not parse HTTP label: {}", err);
```

Comment:
Is there any specific reason why an error decoding from protobuf produces a `warn`, but not from JSON?